measurements/api/measurements.py

```diff
@@ -201,7 +234,10 @@ def get_measurement(measurement_id, download=None):
                 'attachment', filename=filename)
     return response
 
+
 def input_cte(input_, domain, test_name):
+    """Given a domain or an input_, build a WHERE filter
+    """
     if input_ and domain:
         raise BadRequest("Must pick either domain or input")
```

Comment:
So actually what I was saying before doesn't seem to be accurate I can see now from here.

I suppose then you can ignore my previous two comments as it actually only takes `input_` OR `domain`.

We probably don't actually want to support filtering by either of the two as it will likely lead to increased query time.

Reply:
This was the previous behavior of the API, I just added a comment.